### ISSUE TITLE
Add a reconnect button for client

### ISSUE THREAD

**Issue by snippymirror** [AttorneyOnline/AO2-Client#909]
#### **Is your feature request related to a problem? Please describe.**
It is not uncommon to experience disconnections while using AO2, and it is also common to experience multiple disconnections in a row. Unfortunately, as of right now the only way to rejoin is to go back to the masterlist or favorites and rejoin from there.

#### **Describe the solution you'd like**
Perhaps it could be beneficial to incorporate a feature that enables seamless reconnection to a previous server connection. One possible solution could be the addition of a button on the disconnect popup for easy access.

#### **Additional context**
Sal asked me to share this post. :^)

### **TLDR;**
Add a method of reconnecting to the server after disconnection without having to re-enter the masterlist.

**Comment by stonedDiscord** [comment]
i find it very uncommon to be disconnected
i'd say most times this happens it is intentional - through a kick for example
